docs/components/movement-sensor/_index.md

```diff
@@ -0,0 +1,675 @@
+---
+title: "Movement Sensor Component"
+linkTitle: "Movement Sensor"
+weight: 70
+type: "docs"
+description: "A sensor that measures location, kinematic data, or both."
+tags: ["movement sensor", "gps", "imu", "sensor", "components"]
+icon: "img/components/imu.png"
+no_list: true
+aliases:
+    - /components/movement-sensor/
+# SME: Rand
+---
+
+The movement sensor component is an abstraction of a sensor that gives data on where a robot is and how fast it is moving.
+
+We have chosen to abstract these types of sensors into one common API.
+There are many different types of sensors that can provide data for some or all of the following methods: `Position`, `Orientation`, `LinearVelocity`, `AngularVelocity`, `LinearAcceleration` and `CompassHeadings`.
+A global positioning system (GPS) can provide position, linear velocity and compass headings.
+An inertial measurement unit (IMU) can provide angular velocity and orientation.
+We can further apply algorithms, such as a [Kalman filter](https://en.wikipedia.org/wiki/Kalman_filter), to combine data from both a GPS and an IMU to output the full set of information of the movement sensor methods.

```

Comment:
Avoid first person pronouns in the docs - focus on the user. Put the why first and then the how. (If you need to combine... )